pit30m/data/submap.py

```diff
@@ -20,6 +19,28 @@ def __init__(self, submap_id):
         super().__init__(f"Pose not found for submap {submap_id}")
 
 
+# These logs are known to contain at least one submap for which we do not have UTM.
+LOG_ID_TO_MISSING_SUBMAPS = {
+    # Crash on 2023-01-29 -- 5dedc51d-5bd3-4026-fd93-dabb4744ab23 submap UUID was missing
+    # Visual inspection seems to indicate this is in the city of pittsburgh, so not a highway.
+    "327b7948-4e5f-4d8c-f08b-4fc44a067996": [
+        UUID("5dedc51d-5bd3-4026-fd93-dabb4744ab23"),
+    ],
+    # Found on 2023-02-20 -- 1a8132d0-2634-4ad0-e433-ce5987ef0120 submap UUID was missing
+    "c9e9e7a7-f1cb-4af8-c5c9-3a610cbcc20e": [
+        UUID("1a8132d0-2634-4ad0-e433-ce5987ef0120"),
+    ],
+}
+
+BLANK_UUID = UUID("00000000-0000-0000-0000-000000000000")
+
+# test-query poses have been sanitized from the dataset and are not available. The sanitization includes the submap ID,
+# which for these entries has been replaced wiht th a null UUID.
+KNOWN_MISSING_SUBMAPS = [submap_id for submap_ids in LOG_ID_TO_MISSING_SUBMAPS.values() for submap_id in submap_ids] + [
+    BLANK_UUID
+]
```

Comment:
I find this a bit hard to read. Consider using `list(itertools.chain(*list_of_lists))`, or converting the values of the `LOG_ID_TO_MISSING_SUBMAPS` dict to sets

Reply:
cleaned up with more comments!